### PR TITLE
New Icon: Ton

### LINKED ISSUE #8241: Icon Request: Ton
### Brand Name

Ton (The Open Network)

### Website

https://ton.org/

### Popularity Metric

The SimilarWeb's rank is 88,756
https://www.similarweb.com/website/ton.org/#overview

### Official Resources for Icon and Color

https://ton.org/en/brand-assets

### Additional Comments

_No response_

### Contributing

Yes

### PR DESCRIPTION
**Pull request by ripzerotwo**
**Issue:** closes #8241

**Similarweb rank:** 88,756 (https://www.similarweb.com/website/ton.org/#overview)

### Checklist
  - [X] I updated the JSON data in `_data/simple-icons.json`
  - [X]  I optimized the icon with SVGO or SVGOMG
  - [X] The SVG `viewbox` is `0 0 24 24`

### Description
ton

### CODE REVIEW

File: icons/ton.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Ton</title><path d="M8.4 9c-.1 0-.2.1-.2.2s-.1.1 0 .2c0 0 0 .1.1.3l3.2 5.6V9H8.4zm7.3.1c0-.1 0-.1 0 0-.1-.1-.2-.1-.3-.1h-2.9v6.3l3.3-5.6c0-.1.1-.2.1-.3 0-.1 0-.2-.1-.2 0-.1-.1-.1-.1-.1zM12 0C5.4 0 0 5.4 0 12s5.4 12 12 12 12-5.4 12-12S18.6 0 12 0zm4.7 10.2-4.3 7.3c-.1.2-.3.3-.5.3s-.4-.1-.5-.3l-4.2-7.3c.1-.2-.1-.4-.1-.7 0-.3 0-.6.2-.8.1-.3.4-.5.6-.6.3-.1.6-.1.7-.1h6.7c.2 0 .5 0 .7.2.3.1.5.4.6.5.1.2.2.5.2.8.1.1 0 .4-.1.7z"/></svg>
```
Comment (service-paradis) [inline icons/ton.svg:1]: I'm seeing major differences between your version and the SVG symbol found in their brand assets. Can you take another look at it?
image

**Review by service-paradis — CHANGES_REQUESTED** [review]

**Comment by T8RIN** [comment]
I can provide normal icon

**Comment by liketurbo** [comment]
@ripzerotwo, Hi there! Sorry to bother you, but could you, please, update the code as per the requested changes? 
Alternatively, if you've lost interest, could you close the PR as only one PR can exist at a time?

File: _data/simple-icons.json [hunk 2]
```diff
@@ -11666,6 +11666,11 @@
             "hex": "000000",
             "source": "https://global.tomorrowland.com/"
         },
+        {
+            "title": "Ton",
+            "hex": "0088CC",
+            "source": "https://ton.org/en/brand-assets"
+        },
```
Comment (PeterShaggyNoble) [inline _data/simple-icons.json:11673]:
```suggestion
        {
            "title": "TON",
            "hex": "0098EA",
            "source": "https://ton.org/en/brand-assets",
            "guidelines": "https://ton.org/en/brand-assets"
        },
```

File: icons/ton.svg (hunk 1 above)
Comment (PeterShaggyNoble) [inline icons/ton.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Ton</title><path d="M12 0C5.373 0 0 5.373 0 12s5.373 12 12 12 12-5.373 12-12S18.627 0 12 0zM7.902 6.697h8.196c1.505 0 2.462 1.628 1.705 2.94l-5.059 8.765a.86.86 0 0 1-1.488 0L6.199 9.637c-.758-1.314.197-2.94 1.703-2.94zm4.844 1.496v7.58l1.102-2.128 2.656-4.756a.465.465 0 0 0-.408-.696h-3.35zM7.9 8.195a.464.464 0 0 0-.408.694l2.658 4.754 1.102 2.13V8.195H7.9z"/></svg>
```

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]

**Review by PratyushJayachandran — APPROVED** [review]
Thank you for finishing up @PeterShaggyNoble and thank you for starting PR @ripzerotwo  🎉

**Pull request merged**